### PR TITLE
💄 Design cleanup

### PR DESCRIPTION
**Pull request by vakareeralil**
## Description
Betterment in UI for signup and login

### CODE REVIEW

File: praas-app-ionic/src/styles/common.scss [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
-}
+
 .error {
   color: var(--ion-color-danger);
   font-size: 12px;
   text-align: left;
 }
+
+/* Utility Classes */
+.u-text-align-center {
```
Comment (sirajulmcb) [inline praas-app-ionic/src/styles/common.scss:22]: can we have a better prefix? something like we have for `ion`?

Comment (vakareeralil) [reply praas-app-ionic/src/styles/common.scss:22]: It is better not to mess up with the ion namespace. Was following the .u style for utility .o style for object and .c style for component

Comment (sirajulmcb) [reply praas-app-ionic/src/styles/common.scss:22]: yeah, i din't mean to use `ion-` prefix. we should be having something like `praas-` than `.o`, `.u` which might end up messy in the end and difficult.

File: praas-app-ionic/src/store/conduit/del.js [hunk 2]
```diff
@@ -19,13 +23,25 @@ export const deleteConduitFailure = (error) => ({
 
 export const deleteConduit = (conduitId) => {
   return (dispatch) => {
+    console.log('here ');
```
Comment (sirajulmcb) [inline praas-app-ionic/src/store/conduit/del.js:26]: don't console.log here. 😛 
```suggestion
```

File: praas-app-ionic/src/store/conduit/del.js [hunk 3]
```diff
@@ -19,13 +23,25 @@ export const deleteConduitFailure = (error) => ({
...
     dispatch({ type: DELETE_CONDUIT_REQUEST });
+    console.log('here');
```
Comment (sirajulmcb) [inline praas-app-ionic/src/store/conduit/del.js:28]:
```suggestion
```

File: proxy-server/src/server.js [hunk 4]
```diff
@@ -1,6 +1,9 @@
 const path = require('path');
 const express = require('express');
+<<<<<<< HEAD
+=======
 const bodyParser = require('body-parser');
+>>>>>>> e3860a9670af2ae2dbe6c5c31cccd44550b517a9
```
Comment (sirajulmcb) [inline proxy-server/src/server.js:6]:
```suggestion
const bodyParser = require('body-parser');
```

File: proxy-server/src/server.js [hunk 5]
```diff
@@ -112,6 +115,13 @@ function getProxyServerCredentials() {
 }
 
 async function fetchConduits (user) {
+<<<<<<< HEAD
```
Comment (sirajulmcb) [inline proxy-server/src/server.js:118]: Conflict marker

File: proxy-server/src/server.js [hunk 6]
```diff
@@ -112,6 +115,13 @@ function getProxyServerCredentials() {
...
+  // console.log('logged in?', data);
+  // save our token...
+  global.localStorage.setItem('user', JSON.stringify({ ...user }));
```
Comment (sirajulmcb) [inline proxy-server/src/server.js:121]: why using localStorage in `server.js` ?

**Pull request merged**